app/controllers/api/events_controller.rb

```diff
@@ -0,0 +1,6 @@
+class API::EventsController < ApplicationController
+  def index
```

Comment:
This will be a typical controller.  So let me know if you liked a different approach.